### PR TITLE
Integrate DAO UI

### PR DESCRIPTION
**Pull request by jagracar**
In this PR i will try to include the DAO UI into the teia page

Still a lot of things to be done

Feel free to modify at will

### CODE REVIEW

**Comment by jagracar** [comment]
Hi. All needed things should be there.

The CSS is terrible and should be improved.

It should also use the teia form components in the proposal creation page,  but i'm to confused to use them... Could anyone of you take care of that?

I think from this moment you can start to help me to do things better and nicer. I don't plan to touch this today anymore.

**Comment by jagracar** [comment]
38 files involved in this PR... It's a big one...

But the idea was to refactor it as much as possible to be able to reuse a lot of the code for the future Multisig UI and teia vote UIs.

**Comment by jagracar** [comment]
Hi @melMass @xat @Zir0h 

My work is done. This PR has all that is needed for the DAO UI.
I tried to follow as much as possible the teia-ui code style, but problably i missed a couple of things. 
Feel free to modify the code if you think something could be done better.

**Comment by jagracar** [comment]
Ok, i lied...  
While i was waiting for the review i started to work on the teia polls page. It took me very little to add it, so i decided to combine it with this PR, because they use the same code base and it would be nice to have both pages ready at the same time.

**Comment by melMass** [comment]
> Ok, i lied... While i was waiting for the review i started to work on the teia polls page. It took me very little to add it, so i decided to combine it with this PR, because they use the same code base and it would be nice to have both pages ready at the same time.

I have a similar issue, I told everyone I was free at the same time 😅 !
Let's try to get it merged by next week! I will be able to dedicate a bit of time tomorrow morning and can't really predict when for the rest but I will be able to before next week

**Comment by jagracar** [comment]
> > Ok, i lied... While i was waiting for the review i started to work on the teia polls page. It took me very little to add it, so i decided to combine it with this PR, because they use the same code base and it would be nice to have both pages ready at the same time.
> 
> I have a similar issue, I told everyone I was free at the same time 😅 ! Let's try to get it merged by next week! I will be able to dedicate a bit of time tomorrow morning and can't really predict when for the rest but I will be able to before next week

Great!
I will do some more changes today. I want to add a pages for each individual poll and DAO proposal, like we have for OBJKTs:

teia.art/polls/123
teia.art/dao/123

**Comment by jagracar** [comment]
55 files changed or added. If you let me continue working in this PR i'm sure i can reach 100! XD

**Comment by melMass** [comment]
I'm merging back main now. 
I can start the review tomorrow if it's ready @jagracar ?

**Comment by jagracar** [comment]
> I'm merging back main now. I can start the review tomorrow if it's ready @jagracar ?

Sure. Go for it!

File: src/atoms/input/SimpleInput.jsx
Comment (melMass) [inline src/atoms/input/SimpleInput.jsx:42]: Do you remember why this was needed? I just tried replacing them all with the `Input` one and it seems to work fine but I remember you mentioned having issues with it.

**Review by melMass — APPROVED** [review]
All good for me, the only small improvements I can see but that aren't that critical:

- I think we can merge SimpleInput and Input.
- Some places use "raw" html buttons
- We could extract the tables into a Table component
- Claim should probably be removed now (we are the 21)
- @jagracar has to update the contract urls once redeployed.


But overall it's well organised and can be merged once ready

File: src/atoms/input/SimpleInput.jsx
Comment (jagracar) [reply src/atoms/input/SimpleInput.jsx:42]: I had weird problems when i was using a map to create a list of inputs. For example, for the list of addresses in a tez transfer proposal

**Comment by jagracar** [comment]
Thank you for the review, @melMass ! I think we could remove or update the claim page in another PR. We probably should also remove the banner that announced the claim process.

**Comment by melMass** [comment]
> I think we could remove or update the claim page in another PR. 

Unless it becomes too complex I would do it here

> We probably should also remove the banner that announced the claim process.

Yep that's dealt with there, merch just did 😋 https://github.com/teia-community/teia-status/commit/5f053ab4d21a11adcad4d23fbdee3ff2990b0c1e

File: src/atoms/input/SimpleInput.jsx
Comment (melMass) [reply src/atoms/input/SimpleInput.jsx:42]: I see let's not break it now it could be refactored later!
Very very good job overall 👏

**Comment by jagracar** [comment]
Working on it now. Should I remove everything? Or just the direct links to the claim page? It could be useful to keep the old code for future claim processes

**Comment by melMass** [comment]
> Working on it now. Should I remove everything? Or just the direct links to the claim page? It could be useful to keep the old code for future claim processes

Yep I would keep it all just remove it's direct access from the menu

**Comment by jagracar** [comment]
I redeployed the contracts and updated the UI. I think we can probably merge this tomorrow.

**Review by Zir0h — APPROVED** [review]

**Review by melMass — APPROVED** [review]

**Comment by jagracar** [comment]
We will probably merge it tomorrow. I just need to click the "Squash and merge" button, right?

**Comment by melMass** [comment]
> We will probably merge it tomorrow. I just need to click the "Squash and merge" button, right?

It's maybe better to rebase merge this one, it will be a massive commit otherwise

**Pull request merged**
